quickstarts/aws/aws-textract/config.yml

```diff
@@ -0,0 +1,32 @@
+slug: aws-textract
+description: |-
+  ## Monitor your AWS Textract Metrics
+
+  With New Relic's Amazon Textract integration, you now can see all your Textract data on New Relic Instant Observability Platform.
+
+  ### Get started!
+
+   Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-textract/) to learn more about New
+   Relic monitoring and integration with Amazon Textract.
+
+summary: Use New Relic - Amazon Textract Integration to monitor your Textract utilizing Apps and Services with New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Amazon Textract
+documentation:
+  - name: Amazon Textract installation docs
+    description: |
+      Monitor Amazon Textract by connecting AWS to New Relic.
+    url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-textract/
+keywords:
+  - aws
+  - amazon web services
+  - textract
+installPlans:
+  - aws-textract
+dataSourceIds:
```

Comment:
This data source does not exist. Please create one so that the quickstart can properly refer to it. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources

Reply:
added